site/components/header/small-screen-nav/index.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+
+import React from 'react';
+
+import styles from './style.css';
+
+const NavItem = ({ href, tabIndex, text }) => (
+  <li><a href={href} tabIndex={tabIndex}>{text}</a></li>
+);
+
+const { number, string } = React.PropTypes;
+NavItem.propTypes = {
+  href: string.isRequired,
+  tabIndex: number.isRequired,
+  text: string.isRequired,
+};
+
+export default class SmallScreenNav extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      navOpen: false,
+    };
+  }
+
+  handleInputChange = event => {
+    this.setState({
+      navOpen: event.target.checked,
+    });
+  }
+
+  closeMenu = () => {
+    this.input.checked = false;
+  }
+
+  render() {
+    const { navOpen } = this.state;
+    const navItems = [
```

Comment:
We can't do this as some of them have to be a href and some have to be Link with to.

This breaks the link to the what-we-do page